src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -68,6 +68,11 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             // Ensure the HTTP binding routing is registered after all middleware
             builder.UseHttpBindingRouting(applicationLifetime, routes);
 
+            if (environment.IsLinuxConsumption())
+            {
+                builder.UseCors(ScriptConstants.CorsPolicyName);
```

Comment:
this will register the CORS middleware at the application level, which shouldn't be required, correct?

Reply:
You're right. Somehow I had thought the middleware wasn't working without this piece, but it turns out to be okay.